buildenv/jenkins/common/build.groovy

```diff
@@ -404,6 +416,13 @@ def archive_sdk() {
                 env.CUSTOMIZED_SDK_URL = "${ARTIFACTORY_CONFIG[ARTIFACTORY_CONFIG['defaultGeo']]['url']}/${ARTIFACTORY_CONFIG['uploadDir']}${SDK_FILENAME}"
                 currentBuild.description += "<br><a href=${CUSTOMIZED_SDK_URL}>${SDK_FILENAME}</a>"
 
+                if (!zelixFiles.isEmpty()) {
+                    for (String zelixFile in zelixFiles) {
+                        def zelixFileStripped = zelixFile.substring(zelixFile.lastIndexOf("/") + 1)
```

Comment:
> I think this removes too much. I think we only want to remove the `make/` prefix, so, for example, `make/ObfuscateLog.txt` becomes `ObfuscateLog.txt` and `make/whatever/ObfuscateLog-extra.txt` becomes `whatever/ObfuscateLog-extra.txt`. That is, unless the spec added above yields an artifactory URL with just the simple name

No, by doing it this way it will link to the proper artifactory URL...
`https://na.artifactory.swg-devops.com/artifactory/sys-rt-generic-local/hyc-runtimes-jenkins.swg-devops.com/Build_JDK11_s390x_zos_Personal/2739/ObfuscateLog-IBMZSecurity.txt`

If we change it, it would look like this which is incorrect...
`https://na.artifactory.swg-devops.com/artifactory/sys-rt-generic-local/hyc-runtimes-jenkins.swg-devops.com/Build_JDK11_s390x_zos_Personal/2739/make/whatever/ObfuscateLog-IBMZSecurity.txt`

Reply:
![image](https://user-images.githubusercontent.com/30191418/166964945-dbaf5e71-728a-40e0-9d4c-bcc647467a59.png)

Reply:
It shouldn't include the `/whatever/` segment, just like how the `.pax` file and `.tar.gz` file are extracted even tho they're in nested directories. It appears to just grab the file that was given by the `"pattern"` identifier.


![image](https://user-images.githubusercontent.com/30191418/166967709-75b9e0b8-a3ed-4c7e-9e6e-d1be68062911.png)

![image](https://user-images.githubusercontent.com/30191418/166967856-81ea9a9a-dcf7-45bf-b8e7-a978fa0d4a0e.png)

Reply:
We're not using the 'flat' property; it defaults to 'false' (see [1]).

Perhaps @AdamBrousseau can explain why `${OPENJDK_CLONE_DIR}` is apparently not reflected in the resulting URLs.

[1] https://www.jfrog.com/confluence/display/JFROG/Using+File+Specs

Reply:
@keithc-ca I think you're looking at the download spec schema. This would be the upload spec schema which `flat` defaults to `true`
>flat | [Default: true]If true, artifacts are uploaded to the exact target path specified and their hierarchy in the source file system is ignored.If false, artifacts are uploaded to the target path while maintaining their file system hierarchy.

Reply:
Thanks. Yes, I was looking at the download section.